languages/php/generic/php_to_generic.ml

```diff
@@ -215,7 +215,11 @@ and opt_expr_to_label_ident = function
   | None -> G.LNone
   | Some e -> (
       match e with
-      | Int (Some i, tok) -> G.LInt (i, tok)
+      | Int pi ->
+          pi
+          |> Parsed_int.bind (fun (i64, tok) ->
```

Comment:
I think those bind and Option.value are not really helping.
I'd rather have some explicit match and having slightly longer code than trying to be smart.

Reply:
Also why we do this new code here?

Reply:
yeah, agree. it's better once i make the type transparent.